### PR TITLE
Add 2015 NYC Developer Summit

### PR DESCRIPTION
**Pull request by ajpiano**

### CODE REVIEW

File: pages/2015/developer-summit.html [hunk 1]
```diff
@@ -0,0 +1,364 @@
+<script>{
+  "title": "jQuery Foundation Developer Summit 2015: New York, NY",
```
Comment (arthurvr) [inline pages/2015/developer-summit.html]: Wrong indentation, this should use tabs like every file :)

File: pages/2015/developer-summit.html [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+  "pageTemplate": "page-blank.php",
+  "customFields": [
+    {
+      "key": "stylesheets",
+      "value": [
+        "http://yui.yahooapis.com/pure/0.2.1/base-min.css",
+        "http://fonts.googleapis.com/css?family=Raleway:700,400|Open+Sans:400italic,400,800,700",
```
Comment (arthurvr) [inline pages/2015/developer-summit.html]: > http

Switch to https? This wouldn't work when loading the site over https.

File: pages/2015/developer-summit.html [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+        "//code.jquery.com/jquery-1.11.3.js",
+        "/resources/2015/developer-summit/js/scripts.js"
+      ]
+    }
+  ]
+}</script>
+
+<!-- This makes HTML5 elements work on IE 8 and earlier -->
+<script>
+  document.createElement("section");
```
Comment (arthurvr) [inline pages/2015/developer-summit.html]: JavaScript style: `document.createElement( "section" );` (spacing)

But maybe this code just shouldn't live here and we should use a shim.

File: pages/2015/developer-summit.html [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+</script>
+
+<nav id="menu" role="custom-dropdown">
+  <div class="container">
+
+    <input type="checkbox" id="button">
+    <label for="button" onclick=""></label>
+
+    <ul>
+      <li><a class="nav-btn" href="#home">HOME</a></li>
```
Comment (arthurvr) [inline pages/2015/developer-summit.html]: > `HOME`

I'm wondering if we should do the uppercasing of these items with CSS instead.

File: resources/2015/developer-summit/js/scripts.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
+$(function() {
```
Comment (arthurvr) [inline resources/2015/developer-summit/js/scripts.js]: Can you double check this file to use the JavaScript style guide? I'm seeing a lot of spacing issues and some single quotes.

File: Gruntfile.js [hunk 6]
```diff
@@ -9,6 +10,15 @@ grunt.initConfig({
 	"build-resources": {
 		all: "resources/**"
 	},
+	watch: {
+		all: {
+			files: ['Gruntfile.js', 'pages/**/*.html', 'resources/**/*.css'],
```
Comment (arthurvr) [inline Gruntfile.js]: use the JavaScript style guide ;) (quotes and spacing)

File: pages/2015/developer-summit.html [hunk 7]
```diff
@@ -0,0 +1,364 @@
...
+</section>
+
+<section id="conduct">
+  <div class="container">
+    <h1>Conduct</h1>
+    <p>The jQuery Foundation believes that everyone deserves a thoroughly pleasant event experience, regardless of who they are. We adhere to the <a href="https://jquery.org/conduct/" target="_blank">jQuery Foundation Code of Conduct</a> and expect that all of our speakers, attendees, and volunteers will do the same.</p>
+  </div>
+</section>
+
+<script type="text/javascript">
```
Comment (arthurvr) [inline pages/2015/developer-summit.html]: `type` is moot

File: pages/2015/developer-summit.html [hunk 8]
```diff
@@ -0,0 +1,364 @@
...
+
+    <p>Since there are limited spots available, we ask that you fill out this entire application to be considered for a spot. We are looking for individuals from underrepresented groups including — but not limited to — people of color, women, LGBTQIA individuals, etc. who have a strong desire to learning more about open source and making meaningful contributions to open source projects.</p>
+
+    <p>Applications to the Developer Summit close on <strong>Sunday, September 27th</strong> and participants will be notified by <strong>Monday, October 5th</strong>.</p>
+
+    <p>While the event is free to all chosen participants, we cannot provide assistance with travel and accommodation for individuals traveling to the event outside of New York City.</p>
+
+    <p>Got questions for us before you apply? Get in touch with us on Twitter at <a href="https://twitter.com/jqcon">@jqcon</a> or e-mail us at <a href="mailto:content@jquery.org">content@jquery.org</a>.
```
Comment (arthurvr) [inline pages/2015/developer-summit.html]: Not sure, is the mail address correct? Isn't there a mail specifically for the events?

File: Gruntfile.js [hunk 9]
```diff
@@ -30,5 +40,6 @@ grunt.registerTask( "build-events", "Generate events.json resource", function()
 });
 
 grunt.registerTask( "build", [ "build-posts", "build-resources", "build-events" ] );
+grunt.registerTask( "default", [ "deploy" ] );
```
Comment (arthurvr) [inline Gruntfile.js]: I honestly don't think this is really needed, and we don't do this at any other site eighter.

File: resources/2015/developer-summit/css/main.css [hunk 10]
```diff
@@ -0,0 +1,1016 @@
...
+					Subheader
+======================== */
+
+#subheader {
+	background-image: url(../img/bg-subheader.jpg);
+	background-position: center center;
+	background-repeat: no-repeat;
+	-webkit-background-size: cover;
+	-moz-background-size: cover;
+	-o-background-size: cover;
```
Comment (arthurvr) [inline resources/2015/developer-summit/css/main.css]: The `-o-` prefix seems moot here. Only Opera 10.1 ever used this.

File: resources/2015/developer-summit/css/main.css [hunk 11]
```diff
@@ -0,0 +1,1016 @@
...
+	background-color: #111a27;
+	color: #fff;
+}
+
+#program .event-entity .block-type.break img {
+	vertical-align: middle;
+	width: 100px;
+}
+
+#program .speaker_entity {
```
Comment (arthurvr) [inline resources/2015/developer-summit/css/main.css]: Is there a reason it `event-entitiy` (with a dash) and `speaker_entity` (with an underscore). Seems more like an inconsistency.

File: resources/2015/developer-summit/css/main.css [hunk 12]
```diff
@@ -0,0 +1,1016 @@
...
+	right: 0;
+	position: absolute;
+}
+
+#call:after {
+	background-image: url(../img/bg-people.jpg);
+}
+
+#register:after {
+	background-image: url(../img/bg-people-2.jpg);
```
Comment (arthurvr) [inline resources/2015/developer-summit/css/main.css]: Seems 404-ing.

File: resources/2015/developer-summit/css/main.css [hunk 13]
```diff
@@ -0,0 +1,1016 @@
+body {
+	font-family: 'Open Sans', sans-serif;
```
Comment (arthurvr) [inline resources/2015/developer-summit/css/main.css]: Use the CSS style guide, double quotes :p Found some other similar stylistic problems in this file.

File: resources/2015/developer-summit/css/main.css [hunk 14]
```diff
@@ -0,0 +1,1016 @@
...
+	color: white;
+}
+
+#call:after, #register:after {
+	content: "";
+	background-position: center center;
+	background-repeat: no-repeat;
+	-webkit-background-size: cover;
+	-moz-background-size: cover;
+	-o-background-size: cover;
```
Comment (arthurvr) [inline resources/2015/developer-summit/css/main.css]: This one is [moot](http://caniuse.com/#search=background-size), no?

File: pages/2015/developer-summit.html [hunk 15]
```diff
@@ -0,0 +1,364 @@
...
+        <div class="keynote block-type">Social Event</div>
+        <div class="speaker_entity">
+          <div>
+            <p>After two day of hard work, come out, unwind, and wax poetic about the future (and the weekend just past) with new friends and colleagues.</p>
+            <p>Details TBD</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    </div>
```
Comment (arthurvr) [inline pages/2015/developer-summit.html]: Those last two lines seem like wrong indentation (or an extra tag)

**Comment by arthurvr** [comment]
Nice work! :)

**Comment by kborchers** [comment]
Thanks for the review @arthurvr. TBH, I am really not concerned with code style, white space, etc on this site. If it works across browsers and mobile I say ship it.

**Comment by arthurvr** [comment]
Feel free to ignore the style comments, it's up to you :) I don't see why we'd be making exceptions for this site though.

**Comment by ajpiano** [comment]
There's a fair bit of time-sensitivity to getting the site out for a soft-launch over the weekend. I really appreciate the thorough review though @arthurvr - a lot of these issues would be ideal nuggets for folks to work on during the event :)

**Comment by arthurvr** [comment]
I just tried to think about it from a contributor standpoint, we generally ask people people to have every single space right, but then ignore a bunch of those things ourselves. Anyways, I'm not going to waste too much time nitpicking, so go for it :p Some other things are worth resolving, though, such as the mixed content.

**Comment by kborchers** [comment]
@rubymorillo signed the CLA so not sure why it says she didn't so I'm going to merge this

**Comment by jzaefferer** [comment]
Both email and name are in all uppercase on the CLA signature, but regular case on the commit. The CLA is case-sensistive.

Instead of just ignoring the failed check, you could look at the details or ping me about it.

**Comment by kborchers** [comment]
The details said it wasn't signed which wasn't true. Case sensitivity on the check seems like major overkill since an email to kris@jquery and KRIS@jquery goes to the same place

**Comment by jzaefferer** [comment]
Then [file an issue](https://github.com/jquery/jquery-license/issues) to make the check case insensitive. No one claimed the automatic check is perfect, but ignoring it doesn't get us anywhere.

**Pull request merged**